defaults.json

```diff
@@ -296,13 +619,88 @@
         "Weasel",
         "Weese",
         "Wenda",
+        "Werlag",
         "Wilbert",
-        "Will Humble",
         "Will",
+        "Will Humble",
+        "Willam",
         "William",
+        "Willit",
+        "Willum",
+        "Wolmer",
         "Woth",
+        "Wulfe",
+        "Wylla",
         "Yandry",
         "Yna",
-        "Zei"
-    ]
+        "Yorkel",
+        "Ysilla",
+        "Zei",
+        "Zekko",
+        "Zollo"
+    ],
+
+    "sentimentlist": {
+        "honorable": 2,
+        "nothing": -1,
+        "fck": -4,
+        "fck u": -4,
+        "coolest": 2,
+        "lord": 3,
+        "princess": 3,
+        "boss": 3,
+        "humble": 2,
+        "smug": -2,
+        "feast": 2,
+        "woo hoo": 3,
+        "junk": -2,
+        "independent": 2,
+        "beast": -3,
+        "cutest": 2,
+        "birth": 1,
+        "birthday": 2,
+        "bald": -1,
+        "against": -2,
+        "stand": 2,
+        "gracias": 2,
```

Comment:
Ouh, if it wasn't clear: gracias means "thank you" and that guy is surely not expressing neither happy or unhappy feelings! :laughing: 
I honestly don't think "thank you" has any information on + or - feelings, but **I might be wrong**